financial-circuit-service/futureValue/internal/api/controllers/validation.go

```diff
@@ -0,0 +1,56 @@
+package controllers
+
+import (
+	"futurevalue/internal/api/json"
+	"strings"
+)
+
+// Validation
+func validateProveInput(p string) string {
+	m := "Missing input(s): "
```

Comment:
Would replace these with @require tags in the json structures used to shape requests